### ISSUE TITLE
Support New Intel Kernels

### ISSUE THREAD

**Issue by elibol** [nums-project/nums#106]
See here: https://github.com/IntelPython/dpbench/blob/feature/dist/distributed

Current Status for NumS:
- linreg works
- lbfgs and nbody complete, but getting warning messages about division by zero and others.
  - Triu (nbody) not natively supported.
- jstencil crashes
- Mandelbrot throws assertion error about array shape (_advanced_single_array_subscript).
  - meshgrid not natively supported.

**Comment by fschlimb** [comment]
Any plans of fixing this in the near future?

**Comment by elibol** [comment]
Thanks, Frank. We are looking into testing these kernels and opening more concrete issues within the next week.

**Comment by elibol** [comment]
- lbfgs: The numerical precision issues are in line search and can be ignored. NumPy also generated this warning and handles this properly.
- nbody: We've opened an issue to add support for triu (see #126).
- jstencil does not crash with the latest build.
- The Mandelbrot implementation requires advanced indexing, which is only partially supported. A workaround to generating a mesh is to stack the vector row-wise. We've opened an issue for advanced indexing (see #125).
